packages/components/app/styles/components/side-nav/main.scss

```diff
@@ -66,56 +72,100 @@
 
 .hds-side-nav__menu-toggle-button {
```

Comment:
technically this is not anymore a "menu" toggle button, just a "toggle button". maybe rename this class to `hds-side-nav__toggle-button`  (or `hds-side-nav__collapse-toggle-button` if we want to be pedantic)

Reply:
Renamed in 4571e124df5e4c0ccaa190349f55b30db4484f1a